client/src/App.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Route , Link, Redirect, Switch } from 'react-router-dom';
 import 'typeface-roboto';
-import { withStyles, Page, } from './components/elements';
+import { withStyles, Page, DocumentTitle, } from './components/elements';
```

Comment:
Trailing comma is legal here?

Reply:
Yup, it's legal. It's even recommended in multi-line scenarios for cleaner diff and code manipulation: https://github.com/tc39/proposal-trailing-function-commas. In single line statements, not sure if there is any benefit. I just haven't been paying attention to them.